tools/babel-config.js

```diff
@@ -0,0 +1,21 @@
+// This is the same set of transformations that we will enable in a future
+// official React package
+module.exports = {
```

Comment:
Would it be OK to use `require("babel-core/external-helpers");`? Not sure what you mean by "our own implementation."